tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -559,4 +559,43 @@ public function forUserTheSpaceSharesShouldContainTheseEntries(string $user, str
 
 		Assert::assertSame($should, $fileFound, $assertMessage);
 	}
+
+	/**
+	 *
+	 * @param string $user
+	 * @param string $itemId
+	 * @param string $shareSpaceId
+	 *
+	 * @return ResponseInterface
+	 */
+	public function unMountShare(string $user, string $itemId, string $shareSpaceId): ResponseInterface {
+		return GraphHelper::unmountShare(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$this->featureContext->getActualUsername($user),
+			$this->featureContext->getPasswordForUser($user),
+			$itemId,
+			$shareSpaceId,
+		);
+	}
+
+	/**
+	 * @When user :user unmounts share :share using the Graph API
+	 *
+	 * @param string $user
+	 * @param string $share
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userUnmountsShareUsingTheGraphApi(
+		string $user,
+		string $share
+	):void {
+		$shareSpaceId = ($this->spacesContext->getSpaceByName($user, "Shares"))["id"];
```

Comment:
Shares driveId can be hard-coded somewhere and then we can use that
```suggestion
		$shareSpaceId = ($this->spacesContext->getSpaceByName($user, "Shares"))["id"];
```